packages/mdc-slider/README.md

```diff
@@ -0,0 +1,274 @@
+| `registerBodyInteractionHandler(type: string, handler: EventListener) => void` | Adds an event listener `handler` for event type `type` to the `<body>` element of the slider's document |
+| `deregisterBodyInteractionHandler(type: string, handler: EventListener) => void` | Removes an event listener `handler` for event type `type` from the `<body>` element of the slider's document |
+| `registerResizeHandler(handler: EventListener) => void` | Adds an event listener `handler` that is called when the component's viewport resizes, e.g. `window.onresize`. |
+| `deregisterResizeHandler(handler: EventListener) => void` | Removes an event listener `handler` that was attached via `registerResizeHandler` |
+| `notifyInput() => void` | Broadcasts an "input" event notifying clients that the slider's value is currently being changed. The implementation should choose to pass along any relevant information pertaining to this event. In our case we pass along the instance of the component for which the event is triggered for. |
+| `notifyChange() => void` | Broadcasts a "change" event notifying clients that a change to the slider's value has been committed by the user. Similar guidance applies here as for `notifyInput()`. |
+| `setThumbContainerStyleProperty(propertyName: string, value: string) => void` | Sets a dash-cased style property `propertyName` to the given `value` on the thumb container element. |
+| `setTrackStyleProperty(propertyName: string, value: string) => void` | Sets a dash-cased style property `propertyName` to the given `value` on the track element |
+| `isRTL() => boolean` | True if the slider is within an RTL context, false otherwise. |
+
+#### MDCSliderFoundation API
+
+| Method Signature | Description |
+| --- | --- |
+| `layout() => void` | Same as layout() detailed within the component methods table. Does the majority of the work; the component's layout method simply proxies to this. |
+| `getValue() => number` | Returns the current value of the slider |
+| `setValue(value: number) => void` | Sets the current value of the slider |
+| `getMax() => number` | Returns the max value the slider can have |
+| `setMax(max: number) => void` | Sets the max value the slider can have |
+| `getMin() => number` | Returns the min value the slider can have |
+| `setMin(min: number) => number` | Sets the min value the slider can have |
+| `getStep() => number` | Returns the step value of the slider |
+| `setStep(step: number) => void` | Sets the step value of the slider |
+| `isDisabled() => boolean` | Returns whether or not the slider is disabled |
+| `setDisabled(disabled: boolean) => void` | Disables the slider when given true, enables it otherwise. |
+
+### Theming
+
+All thematic elements of sliders make use of the **primary theme color**, including when the
```

Comment:
Yes, but we document that directly within the demo itself. Do you think it's not called out enough?